packages/main/src/index.ts

```diff
@@ -1,10 +1,13 @@
 // Modules to control application life and create native browser window
-import { app, BrowserWindow, ipcMain } from "electron"
+import { app, BrowserWindow, ipcMain, protocol } from "electron"
 import contextMenu from "electron-context-menu"
 import isDev from "electron-is-dev"
 import { createIpcExecutor, createSchemaLink } from "graphql-transport-electron"
 import schema from "./schema"
-
+import * as cache from "./cache"
+import { parseBodyUri } from "poodle-common/lib/models/uri"
+import { contentType, filename } from "./models/MessagePart"
+import { PassThrough } from "stream"
```

Comment:
By the way, I like to keep imports sorted in a certain way. The atom-typescript plugin adds the `typescript:organize-imports` command that sorts imports automatically, and removes unused imports. You can run it by bringing up the command palette with Cmd+Shift+P, and selecting the "Organize module imports" command.